save-common/src/jvmMain/kotlin/org/cqfn/save/entities/TestExecution.kt

```diff
@@ -29,17 +28,13 @@ class TestExecution(
 
     @ManyToOne
     @JoinColumn(name = "agent_id")
-    var agent: Agent,
-
-    @ManyToOne
-    @JoinColumn(name = "project_id")
-    var project: Project,
+    var agent: Agent?,
```

Comment:
Please add in kdocs that this field is nullable because agents are not created yet by the moment of test execution creation